tests/python/relay/aot/test_c_device_api.py

```diff
@@ -151,7 +158,7 @@ def test_device_api_hooks_unpacked_api(device_api_main_func):
     # We dont need to check exact input and output var names in this test.
     # Hence, using a regex to cover any legal I/O name.
     regex = re.compile(
-        'tir\.tvm_check_return\(0, -1, tir\.call_extern\("tvmgen_default_ethos_u_main_0", \w+, \w+, device_context_ethos_u\)\)'
+        r'tir\.tvm_check_return\(0, -1, tir\.call_extern\("tvmgen_default_ethos_u_main_0", \w+, \w+, device_context_ethos_u\)\)'  # pylint: disable=line-too-long
```

Comment:
could you break this up rather than disabling? e.g.
```
r'tir\.tvm_check_return\('
r'0, -1, '
r'tir\.call_extern\("tvmgen_default_ethos_u_main_0", '
r'\w+, \w+, device_context_ethos_u\)\)'
```